packages/flutter/lib/src/material/bottom_navigation_bar_theme.dart

```diff
@@ -124,6 +126,11 @@ class BottomNavigationBarThemeData with Diagnosticable {
   /// If non-null, overrides the [BottomNavigationBar.landscapeLayout] property.
   final BottomNavigationBarLandscapeLayout? landscapeLayout;
 
+  /// {@macro flutter.material.bottomnavigationbar.mouseCursor}
```

Comment:
This line can be deleted. The line below, "If specified, overrides...", is sufficient.